cardano-rt-view/cardano-rt-view.cabal

```diff
@@ -0,0 +1,69 @@
+name:                  cardano-rt-view
+version:               1.0.0
+description:           Real-time view service for cardano node
+author:                IOHK
+maintainer:            operations@iohk.io
+license:               Apache-2.0
+license-files:
+  LICENSE
+  NOTICE
+build-type:            Simple
+cabal-version:         >= 1.10
+extra-source-files:    README.md, ChangeLog.md
```

Comment:
we do not want to maintain a Changelog :-)

Reply:
Done.